staging/src/k8s.io/component-base/cli/flag/sectioned.go

```diff
@@ -58,6 +58,17 @@ func (nfs *NamedFlagSets) FlagSet(name string) *pflag.FlagSet {
 	return nfs.FlagSets[name]
 }
 
+// MergeNamedFlagSets merges the given NamedFlagSets into a new NamedFlagSets instance with all flags combined.
+func MergeNamedFlagSets(nfss ...NamedFlagSets) NamedFlagSets {
+	newNFS := NamedFlagSets{}
+	for _, nfs := range nfss {
+		for _, flagName := range nfs.Order {
```

Comment:
Technically this is a flag SET name, right?  If so, then maybe `flagSetName` would be a more accurate variable name to use, especially since there are flags contained within a flag set.

Reply:
Good point, done.